src/main/java/frc/robot/subsystems/LanuchMechanisumSubsystem.java

```diff
@@ -4,24 +4,54 @@
 
 package frc.robot.subsystems;
 
-import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.REVPhysicsSim;
+
+import edu.wpi.first.math.system.plant.DCMotor;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 
 public class LanuchMechanisumSubsystem extends SubsystemBase {
-    PWMSparkMax topLaunch;
-    PWMSparkMax bottomLaunch;
+    CANSparkMax top;
+    CANSparkMax bottom;
+    CANSparkMax staging;
+    CANSparkMax pickup;
 
   /** Creates a new LanuchMechanisumSubsystem. */
   public LanuchMechanisumSubsystem() {
 
-  topLaunch = new PWMSparkMax(Constants.PWM.topLaunch);
-  bottomLaunch = new PWMSparkMax(Constants.PWM.bottomLaunch);
+    top = new CANSparkMax(Constants.CANSparkMaxID.topRight, MotorType.kBrushless);
+    bottom = new CANSparkMax(Constants.CANSparkMaxID.topLeft, MotorType.kBrushless);
+    staging = new CANSparkMax(Constants.CANSparkMaxID.staging, MotorType.kBrushless);
+    pickup = new CANSparkMax(Constants.CANSparkMaxID.launcherLoad, MotorType.kBrushless);
+  
+
+    bottom.follow(top);
+    staging.follow(pickup);
+
 
+      REVPhysicsSim.getInstance().addSparkMax(top, DCMotor.getNEO(1));
+      REVPhysicsSim.getInstance().addSparkMax(bottom, DCMotor.getNEO(1));
+      REVPhysicsSim.getInstance().addSparkMax(staging, DCMotor.getNEO(1));
+      REVPhysicsSim.getInstance().addSparkMax(pickup, DCMotor.getNEO(1));
   }
+public void launchSpeaker(){
+  top.set(Constants.Misc.speekerLaunchSpeed);
+  pickup.set(Constants.Misc.speekerLaunchSpeed);
+}
+
+public void launchAmp(){
+  //launch into the amp
+  top.set(Constants.Misc.ampLaunchSpeed);
```

Comment:
you probably want to set `pickup` to run at `0` speed with this method, since the mechanism should only be running one set of motors at a time, based on my reading

Reply:
#33